vmdk_plugin/Makefile

```diff
@@ -83,12 +83,14 @@ GO := GO15VENDOREXPERIMENT=1 go
 FPM := fpm
 
 # make sure we rebuild of vmkdops or Dockerfile change (since we develop them together)
-VMDKOPS_MODULE := vmdkops
+VMDKOPS_MODULE := drivers/vmdk/vmdkops
+VMDKOPS_TEST_MODULE := vmdkops
 VMDKOPS_MODULE_SRC = $(VMDKOPS_MODULE)/*.go $(VMCI_SRC)
 
 # All sources. We rebuild if anything changes here
-SRC = plugin.go main.go log_formatter.go refcnt.go \
-	utils/fs/fs.go utils/config/config.go utils/TestInputParamsUtil.go
+SRC = main.go log_formatter.go utils/refcount/refcnt.go \
```

Comment:
The test file needs to be included in the build. May be test code should get separated out later away from the product code.